### ISSUE TITLE
Plans: Poor contrast on subheadings

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#35095]
<img width="324" alt="screen-shot-2019-07-08-at-1 08 04-pm" src="https://user-images.githubusercontent.com/2124984/62386936-42c3f000-b527-11e9-9340-018ff5251b36.png">

#### Steps to reproduce
1. Starting at URL: http://wordpress.com/plans/
2. View subheadings on each plan, as in the image above; note the lack of contrast between foreground text and white background.
3. These subheadings can probably use the CSS var `--color-text-subtle` for adequate contrast.
